src/lib/components/ListItemAction.svelte

```diff
@@ -0,0 +1,55 @@
+<div class="container">
+  <div class="start-wrapper">
+    <div class="icon-wrapper">
+      <slot name="icon" />
```

Comment:
the icon is mandatory or optional? if mandatory, should it be a property?

Just asking in case it would make the component usage easier?

Reply:
Gotcha